test/cmdline_test.py

```diff
@@ -140,7 +140,7 @@ def test_cmdline_logger(self, setup_mock, warn):
 
         with mock.patch("luigi.configuration.get_config") as getconf:
             getconf.return_value.get.side_effect = ConfigParser.NoOptionError(section='foo', option='bar')
-            getconf.return_value.get_boolean.return_value = True
+            getconf.return_value.getint.return_value = 0
```

Comment:
@gf-atebbe may clarify why the True was relevant, apparently it has become irrelevant (get_boolean doesn't exist anymore), yet the MagicMock existence has become relevant with this patch...